internal/charmstore/store.go

```diff
@@ -252,6 +258,7 @@ func (s *Store) insertEntity(entity *mongodoc.Entity) (err error) {
 			Read:  readPerm,
 			Write: writePerm,
 		},
+		Promulgated: false,
```

Comment:
Has been time since I red the spec, but shouldn't this be set to true if the given entity has a promulgated URL?